### ISSUE TITLE
Testing Maps: Saving maps as Dashboard fails

### ISSUE THREAD

**Issue by Palakp41** [influxdata/ui#1003]
As a user, when view data on Data Explorer using Maps and I try to `Save As Dashboard`, Map type fails to load the dashboard with an influx db error. This issue is to investigate if it is a front end code missing or backend go code.

**Comment by Palakp41** [comment]
Upon investigating this, I found that the code that allows saving dashboard from this pr by Bonitoo: https://github.com/influxdata/influxdb/pull/19811 was merged into Influxdb but never pushed over to idpe. This is why saving maps as dashboards or view map dashboards is failing. Updating this story to be blocked on the idpe issue: https://app.zenhub.com/workspaces/applicationui-5e3b05772922e316b3a210a4/issues/influxdata/idpe/10259

Also, I will reword this story that tracks the testing of dashboards after idpe issue is fixed.

**Comment by Palakp41** [comment]
Tested in staging after https://github.com/influxdata/idpe/pull/10261 was merged which adds the backend GeoViewProperties for saving dashboard. Everything is working correctly again. 
![Kapture 2021-03-31 at 14 11 53](https://user-images.githubusercontent.com/26464594/113212060-1fd16e00-922b-11eb-845e-2cfda1b3432e.gif)
